### ISSUE TITLE
Forbid unused `noqa` comments

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#165]
# Rule request

## Thesis

We should raise a warning when `noqa` comment is unused.
However, I am not sure that this is possible without patching `flake8`.

## Reasoning

Consider this example:

```python
some_variable = 12  # noqa: A001
```

`A001` means that your name overrides some `python` builtin.
But, it does not look like so. Or does it?

This kind of errors happen to have historical roots. At some point in time they were actually used, but then the code was refactored. But the comment was left untouched.

**Comment by sobolevn** [comment]
See https://github.com/snoack/flake8-per-file-ignores/blob/master/flake8_per_file_ignores.py#L61-L65

This is already implemented. We just need to copy this logic.

**Comment by sobolevn** [comment]
Related: https://github.com/asottile/yesqa

**Comment by sobolevn** [comment]
We can also warn about unused `ignore=` and `per-file-ignores=`

**Comment by maltevesper** [comment]
I think there is another related option: https://pypi.org/project/flake8-noqa/

**Comment by aleksey-sergey** [comment]
> I think there is another related option: https://pypi.org/project/flake8-noqa/
It's a nice package indeed. Added it to one of my projects and it "just worked"!

**Comment by vnmabus** [comment]
I am really interested in this. It is also useful to remove the silencing of false positives when they are fixed.

**Comment by urtow** [comment]
@vnmabus May be just use flake8-noqa?
